projects/sonar/src/app/record/document/detail/detail.component.html

```diff
@@ -352,12 +353,35 @@ <h5 id="other-files" class="mt-5" translate>Other files</h5>
     <div class="row">
       <ng-container *ngFor="let file of filteredFiles; let first = first">
         <div class="col-lg-2 mb-5" *ngIf="!first">
-          <sonar-document-file [file]="file" (previewClicked)="showPreview($event)"></sonar-document-file>
+          <sonar-document-file [file]="file" [statistics]="record.statistics"
+           (previewClicked)="showPreview($event)"></sonar-document-file>
         </div>
       </ng-container>
     </div>
   </ng-container>
 
+  <ng-container *ngIf="record.statistics">
+    <h5 id="stats" class="mt-5" translate>Statistics</h5>
+    <hr class="mb-4 mt-0">
+    <div class="container pb-5 pl-0">
```

Comment:
Do you mean `row` instead of `container`?

Reply:
Shouldn't `*ngIf` be in `ng-container` rather than `<div class="row">` ?